openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -691,45 +690,6 @@ def check_and_reset_signal():
             lambda: cohorts.add_user_to_cohort(first_cohort, "non_existent_username")
         )
 
-    @patch("openedx.core.djangoapps.course_groups.cohorts.tracker")
-    def add_user_to_cohorts_race_condition(self, mock_tracker):
```

Comment:
This wasn't actually used anymore, and was the only code using the `before_after` package.